blueprint/src/chapters/chapter1/sheet1.tex

```diff
@@ -0,0 +1,23 @@
+\section{Defining the Group}
+
+\begin{definition}[Group]
+    \label{definition : Group}
+    \lean{AlgebraInLean.Group}
+    \leanok
+    \uses{definition : Monoid}
+    A group is a set $G$ with the following properties:
+    \begin{itemize}
+        \item It must have a binary operation $\mu$.
```

Comment:
I'd write the full function signature for `\mu`, i.e.

```
\mu : G \rightarrow G \rightarrow G
```

Do the same for `\iota`.